package.json

```diff
@@ -26,7 +26,8 @@
     "dompurify": "^2.0.11",
     "lodash-es": "^4.17.15",
     "react": "^16.13.1",
```

Comment:
- `react` should be both in `devDependencies` and `peerDepenencies` because we depend on it for local development directly (e.g. `import React from "react"`) and consumers should have it in their project.
- `react-dom` should be a `peerDependencies` because consumers should have it in their project.
- `react-native` can be a `devDependencies` if we're using it for tests or in  a local project, but likely won't need it listed anywhere at all.